components/Table.js

```diff
@@ -61,6 +62,11 @@ export default function Table({
     setAllFilters(filters);
   }, [filters]);
 
+  const handleClick = row => () => {
```

Comment:
This function name might be a little ambiguous if we also need to handle clicks for other table elements in the future (eg. cells, headers). Perhaps it's worth changing it now, but it's likely not a big deal.

Reply:
Oh that's a good idea! I'll update it